repl/src/main/scala/com/cloudera/livy/repl/SparkRInterpreter.scala

```diff
@@ -133,7 +133,7 @@ class SparkRInterpreter(process: Process, backendInstance: Any, backendThread: T
   implicit val formats = DefaultFormats
 
   private[this] var executionCount = 0
-  override def kind: String = "sparkR"
+  override def kind: String = "sparkr"
```

Comment:
Why to change the kind name ? This may cause incompatibility with previous version.